pkg/handler/connect.go

```diff
@@ -827,11 +827,39 @@ func SshJump(ctx context.Context, conf *util.SshConfig, flags *pflag.FlagSet, pr
 	if err != nil {
 		return
 	}
+
+	serverHost := u.Hostname()
+	serverPort := u.Port()
+	if serverPort == "" {
+		if u.Scheme == "https" {
+			serverPort = "443"
+		} else if u.Scheme == "http" {
+			serverPort = "80"
+		} else {
+			// handle other schemes if necessary
+			err = errors.New("kubeconfig is invalid: wrong protocol")
+			return
+		}
+	}
+	ips, err := net.LookupHost(serverHost)
+	if err != nil {
+		// handle error
+		return
+	}
+
+	if len(ips) == 0 {
+		// handle error: no IP associated with the hostname
```

Comment:
you can generate a new error, like `err = fmt.Errorf("no IP associated with the hostname %s", serverHost)`